src/main/sensors/gyro.c

```diff
@@ -668,54 +738,34 @@ static void gyroInitFilterDynamicNotch(gyroSensor_t *gyroSensor)
 }
 #endif
 
-#if defined(USE_GYRO_FAST_KALMAN)
-static void gyroInitFilterKalman(gyroSensor_t *gyroSensor, uint16_t gyro_filter_q, uint16_t gyro_filter_r, uint16_t gyro_filter_p)
-{
-    gyroSensor->fastKalmanApplyFn = nullFilterApply;
-
-    // If Kalman Filter noise covariances for Process and Measurement are non-zero, we treat as enabled
-    if (gyro_filter_q != 0 && gyro_filter_r != 0) {
-        gyroSensor->fastKalmanApplyFn = (filterApplyFnPtr)fastKalmanUpdate;
-        for (int axis = 0; axis < XYZ_AXIS_COUNT; axis++) {
-            fastKalmanInit(&gyroSensor->fastKalman[axis], gyro_filter_q, gyro_filter_r, gyro_filter_p);
-        }
-    }
-}
-#endif
-
-
-#if defined(USE_GYRO_LPF2)
-
-#if GYRO_LPF2_ORDER_MAX > BIQUAD_LPF_ORDER_MAX
-# error "GYRO_LPF2_ORDER_MAX is larger than BIQUAD_LPF_ORDER_MAX"
-#endif
-
-static void gyroInitFilterLpf2(gyroSensor_t *gyroSensor, int order, int lpfHz)
-{
-    const int gyroFrequencyNyquist = 1000000 / 2 / gyro.targetLooptime;
-    int sections = 0;
-    if (lpfHz && lpfHz <= gyroFrequencyNyquist && order <= GYRO_LPF2_ORDER_MAX) {  // Initialisation needs to happen once samplingrate is known
-        for (int axis = 0; axis < XYZ_AXIS_COUNT; axis++) {
-            const int axisSections = biquadFilterLpfCascadeInit(gyroSensor->biquadLpf2[axis], order, lpfHz, gyro.targetLooptime);
-            sections = MAX(sections, axisSections);
-        }
-    }
-    gyroSensor->biquadLpf2Sections = sections;
-}
+#if GYRO_LPF_ORDER_MAX > BIQUAD_LPF_ORDER_MAX
```

Comment:
This should be close to gyroLowpassFilter_t definition ... ot beginning of file

Reply:
Handled in f639e5b